### ISSUE TITLE
Nesting elements affected

### ISSUE THREAD

**Issue by amyamandalee** [amyamandalee/spaceoddity#1]
@ellysmlly @sehsarah
Can I set an element to only affect some of what's nested inside it?
So if I want to make one this disappear and not another.
E.g.

<div class="everything everything--affect-one">
    <div class="one"></div>
    <div class="two"></div>
</div>

.everything--affect-one {
    display: none;
}

and only have div class "one" disappear and not div class "two"

**Comment by sehsarah** [comment]
Yes that's possible, is this still an issue you're having @amyamandalee? Can you link me to this part of your CSS
